src/main/resources/application.properties

```
@@ -1,4 +1,8 @@
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
-telegram.bot.token=
+spring.datasource.url=jdbc:postgresql://localhost:5432/Bot-telegram
+spring.datasource.username=student
+spring.datasource.password=chocolatefrog
+telegram.bot.token=6662840770:AAGTBCJn5RS18zkAgeeCztzADdmCwZst82c
+telegram.bot.name=bot-telegram
+spring.jpa.hibernate.ddl-auto= update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
Таблицы создаём через ликвибейс, поэтому тут должно быть validate